File: templates/tests-php/BREACH-003.php
<?php

namespace Compliance\Tests;

use PHPUnit\Framework\TestCase;

class PIICategoriesTest extends TestCase
{
    public function testVerifyPIICategoriesDocumented(): void
    {
        $config = Config::load();
        $this->assertNotEmpty($config->dataLifecycle->piiCategories);
    }
}

File: templates/tests-csharp/BREACH-003.cs
using Xunit;

namespace Compliance.Tests
{
    public class PIICategoriesTests
    {
        [Fact]
        public void TestVerifyPIICategoriesDocumented()
        {
            var config = Config.Load();
            Assert.NotEmpty(config.DataLifecycle.PiiCategories);
        }
    }
}
